pom.xml

```diff
@@ -63,6 +63,45 @@
             <artifactId>connect-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <scope>provided</scope>
```

Comment:
Scope should be test.

Reply:
We have some uses out of test scope as well.

Reply:
not sure because we haven't modified the existing codebase, just added IT tests and Container.

Reply:
@Sumanthonnavar this maybe a transitive dependency, so making this as test will result in compilation error.
We can remove this dependency in that case.